advanced/server-extension/README.md

```diff
@@ -389,18 +474,8 @@ through package managers like `pip`.
 The `setup.py` file is the entry point to describe package metadata:
 
 ```py
-# setup.py#L56-L87
-
-setup_args = dict(
-    name=name,
-    version=version,
-    url="https://github.com/jtpio/jupyterlab-extension-examples/",
-    author="JupyterLab",
-    description="A minimal JupyterLab extension with backend and frontend parts.",
-    long_description= long_description,
-    long_description_content_type="text/markdown",
-    cmdclass= cmdclass,
-    packages=setuptools.find_packages(),
+# setup.py#L66-L98
+
```

Comment:
The diff shows some missing parts for this snippet.

Reply:
The complete file is now embedded.